utils/apptainer/def/straglr-philres-1.3.1.def

```diff
@@ -0,0 +1,21 @@
+Bootstrap: docker
```

Comment:
Use
```
Bootstrap: localimage
From: sif/build/ubuntu-22.04.sif
```
similar to `samtools-1.17-patch1`. other images still using the ubuntu docker will be updates when the tools get an update.